### PR TITLE
Add make test

### PR DESCRIPTION
**Pull request by joehillen**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,11 @@
+.PHONY: package test
+
 package:
 	./check.sh
 	git rev-parse HEAD >resources/build.txt
 	lein do clean, test :all, package
+
+test:
+	cd test/repo/no-errors && mvn test
+	-cd test/repo/some-errors && mvn test
```
Comment (dakrone) [inline Makefile]: huh, what does the `-` in `-cd` do?

Comment (drewr) [inline Makefile]: `-` ignores the exit code, which you need because that invocation of `mvn test` will fail.

Comment (dakrone) [inline Makefile]: whoa neat! is that a `make` builtin then?

Comment (drewr) [inline Makefile]: Yep, built right into it!

**Pull request merged**